sklearn/multiclass.py

```diff
@@ -547,35 +594,70 @@ def _more_tags(self):
         """Indicate if wrapped estimator is using a precomputed Gram matrix"""
         return {"pairwise": _safe_tags(self.estimator, key="pairwise")}
 
+    def get_metadata_routing(self):
+        """Get metadata routing of this object.
+
+        Please check :ref:`User Guide <metadata_routing>` on how the routing
+        mechanism works.
+
+        .. versionadded:: 1.4
 
-def _fit_ovo_binary(estimator, X, y, i, j):
+        Returns
+        -------
+        routing : MetadataRouter
+            A :class:`~utils.metadata_routing.MetadataRouter` encapsulating
```

Comment:
Note to reviewes: We should make a check by building the doc in nitpick mode before reviewing.
Otherwise, we will introduce some regression.

```suggestion
            A :class:`~sklearn.utils.metadata_routing.MetadataRouter` encapsulating
```